src/foam/nanos/auth/CapabilityAuthService.js

```diff
@@ -100,6 +100,7 @@ foam.CLASS({
               UserCapabilityJunction ucj = (UserCapabilityJunction) obj;
               if ( ucj.getStatus() == CapabilityJunctionStatus.GRANTED ) {
                 Capability c = (Capability) capabilityDAO.find(ucj.getTargetId());
+                c.setX(x);
```

Comment:
This will reset the context on this AuthService to the callers, is that appropriate?

Reply:
I think this only sets the context on the object, tested and there doesn't seem to be any changes in the context of the authservice

Reply:
yes, sorry, read it wrong. Thanks.